internal/providers/pke/pkeworkflow/delete_aws_pool_activity.go

```diff
@@ -71,8 +72,8 @@ func (a *DeletePoolActivity) Execute(ctx context.Context, input DeletePoolActivi
 	}
 
 	stackInput := &cloudformation.DeleteStackInput{
-		StackName: aws.String(stackName),
-		// ClientRequestToken: aws.String(string(activity.GetInfo(ctx).ActivityID)),
+		StackName:          aws.String(stackName),
+		ClientRequestToken: aws.String(activity.GetInfo(ctx).WorkflowExecution.ID),
```

Comment:
Note: I explicitly highlight this change, because this isn't just a client token request (CRT) value replacement, but also an introduction of using CRT in this context.
I assumed the original value was commented out because activity ID was not a reliable CRT source as it changed between activity retries.